main/storage.py

```diff
@@ -1,10 +1,9 @@
+import cssmin
 import jsmin
 from django.contrib.staticfiles.storage import ManifestStaticFilesStorage
 from django.core.files.base import ContentFile
 from django.utils.encoding import smart_str
 
-import cssmin
```

Comment:
`ruff check .` is happy for me, why is the uv used ruff not?

Reply:
Could it be that you are running a different version of `ruff`? We are installing the latest in the CI:

https://github.com/archlinux/archweb/blob/59229f280bb92fae770bc1efe690d82b71cd6e34/.github/workflows/main.yml#L19

This is one of the benefits of managing dependencies with `uv`, since we are now locking a particular version and making sure we always run that version, without having a separate `requirements` for each group.